### PR TITLE
docs: add docs for schemas as database for oracle

### PR DESCRIPTION
**Pull request by rebelice**

### CODE REVIEW

File: content/docs/how-to/oracle/how-to-sync-schemas-as-database-for-oracle.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+description: Step-By-Step guide about how to sync schemas as database for Oracle
+---
+
+1. Click the `Instance` button on the top of Bytebase main page.
+1. Click the `Add Instance` button on the top left first.
+1. Select `Oracle` as the database type.
+1. Select **`Sync Self`** sync mode.
+1. Fill in the other required fields.
+
+
```
Comment (milasuperstar) [inline content/docs/how-to/oracle/how-to-sync-schemas-as-database-for-oracle.md:12]:
```suggestion
```

**Review by milasuperstar — CHANGES_REQUESTED** [review]

**Review by tianzhou — CHANGES_REQUESTED** [review]
This can be put at https://www.bytebase.com/docs/get-started/instance/

create an oracle section

**Comment by rebelice** [comment]
Updated. PTAL @milasuperstar @tianzhou

**Review by tianzhou — APPROVED** [review]
LGTM

**Pull request merged**